### PR TITLE
Improve abstract http call

### PR DESCRIPTION
**Pull request by StefanoGuerrini**
Reviewers: @GetFeedback/platform-java

### CODE REVIEW

File: kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/integration/KaHPPMockServer.java [hunk 1]
```diff
@@ -40,22 +40,20 @@ public static void closeServer() {
     mockServer.close();
   }
 
-  public static void mockHttpResponse(String body, int statusCode) {
-    mockHttpResponse(body, statusCode, null);
+  public static void mockHttpResponse(String path, String body, int statusCode) {
```
Comment (vadzim-kazak) [inline kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/integration/KaHPPMockServer.java:43]: You could have used here method overloading - keep old _mockHttpResponse(String body, int statusCode)_ method also and call from it a new one with hardcoded "/enrich" path - this way you could avoid a lot of changes in the existing code base.

Comment (StefanoGuerrini) [reply kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/integration/KaHPPMockServer.java:43]: True, but the previous signature didn't have much sense

File: kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/unit/configuration/http/SimpleHttpCallTest.java [hunk 2]
```diff
@@ -135,4 +138,60 @@ private JsonNode getJsonNodeFromFile(String file) throws IOException {
...
+                            HTTP_CALL_PATH+"/${value.payload.id}",
+                            "apiClient",
+                            apiClient,
+                            "responseHandler",
+                            ResponseHandlerRecordUpdate.RECORD_VALUE_REPLACE));
+
+    JsonNode key = getJsonNodeFromFile(FIXTURE_KEY);
+    JsonNode value = getJsonNodeFromFile(FIXTURE_VALUE);
+
+    KaHPPMockServer.mockHttpResponse(HTTP_CALL_PATH+"/"+value.get("payload").get("id").textValue(), value.toString(), 200, "{}");
```
Comment (vadzim-kazak) [inline kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/unit/configuration/http/SimpleHttpCallTest.java:161]: Minor thing: as I can see from the source code around spaces are used in string concatenation expressions, maybe we can add some automatic code autoformatting to deal with such cases.

**Review by vadzim-kazak — APPROVED** [review]

File: kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/unit/configuration/http/SimpleHttpCallTest.java (hunk 2 above)
Comment (StefanoGuerrini) [reply kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/unit/configuration/http/SimpleHttpCallTest.java:161]: Let's discuss it tomorrow morning; I didn't understand your point.

**Pull request merged**